src/pages/Account/name/NameSetting.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import gql from 'graphql-tag'
+import { graphql } from 'react-apollo'
+import { store } from '../../../redux'
+import { showNotification } from '../../../actions/rootActions'
+import EditableInputSetting from '../EditableInputSetting'
+import styles from '../userName/UsernameSettings.module.scss'
+
+const CHANGE_NAME_MUTATION = gql`
+  mutation changeName($value: String!) {
+    changeName(name: $value) {
+      name
+    }
+  }
+`
+
+const validateName = name => {
```

Comment:
`[D-0-1]` For function better to use the `function` declaration. We are using `const` for components, one line functions and variables